### PR TITLE
Stylesheet widget

### LINKED ISSUE #34: Support displaying the current style sheet(s) in the debug gui

### PR DESCRIPTION
**Pull request by mkiael**
Closes #34

### CODE REVIEW

File: browser/gui/app.cpp [hunk 1]
```diff
@@ -353,6 +366,15 @@ void App::run_dom_widget() const {
     ImGui::End();
 }
 
+void App::run_stylesheet_widget() const {
+    ImGui::SetNextWindowPos(
+            ImVec2(0, 70 * static_cast<float>(scale_) + window_.getSize().y / 2.f), ImGuiCond_FirstUseEver);
```
Comment (mkiael) [inline browser/gui/app.cpp:371]: Suggestions are welcome for how to handle the y-position here.

Comment (robinlinden) [reply browser/gui/app.cpp:371]: All of the positions and sizes here are very hacked together. :(

Maybe we can do something cleaner using the imgui tab API? Would reduce the number of windows you end up with by default too. If it works well enough, we could even switch to the tab/window docking branch of imgui if it works well enough.

**Review by robinlinden — APPROVED** [review]
Looks good and worked well when I tested it locally! I was missing this! :P

File: browser/gui/app.cpp (hunk 1 above)
Comment (mkiael) [reply browser/gui/app.cpp:371]: Yeah, that sounds good. I'll leave this as it is for now.

**Pull request merged**